authentication.py

```diff
@@ -0,0 +1,92 @@
+import re
+from passlib.context import CryptContext
+from fastapi import HTTPException, status
+import jwt
+
+from models import User
+from config import get_settings
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+
+def get_hashed_password(password):
+    return pwd_context.hash(password)
+
+
+async def very_token(token: str):
+    '''verify token from login'''
+    try:
+        payload = jwt.decode(token, get_settings().SECRET,
```

Comment:
**Performance Issue**: The jwt.decode function is called with a dynamic secret key fetched from get_settings().SECRET. This can lead to performance issues if get_settings() is not properly cached, as it may involve file I/O or other slow operations. <br> **Fix**: Ensure that get_settings() is cached using an appropriate strategy such as lru_cache to avoid repeated I/O operations. <br> **Code Suggestion**: 
 ```
 @@ -19,2 +19,3 @@
 -        payload = jwt.decode(token, get_settings().SECRET,
 +        settings = get_settings()
 +        payload = jwt.decode(token, settings.SECRET,
 ```